apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -151,13 +168,12 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
                   key={app.slug}
                   app={app}
                   eventType={eventType}
-                  {...shouldLockDisableProps("apps")}
```

Comment:
Removed switch from each app as it referred to the general apps locked state now present in the alert